lightning/src/blinded_path/message.rs

```diff
@@ -74,16 +91,39 @@ impl Writeable for ForwardTlvs {
 	}
 }
 
-impl Writeable for ReceiveTlvs {
-	fn write<W: Writer>(&self, writer: &mut W) -> Result<(), io::Error> {
-		// TODO: write padding
```

Comment:
Let's keep this comment. Also, could you move the new code back to here to keep the blame layer clean?

Reply:
Done! [6ff2e49](https://github.com/lightningdevkit/rust-lightning/commit/6ff2e49123c3135dccea720d7104e6ad5b8f7308)
Thanks for the pointer!